images/descheduler/Dockerfile.rhel7

```diff
@@ -1,8 +1,10 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
 WORKDIR /go/src/github.com/kubernetes-incubator/descheduler
 COPY . .
-RUN make build
+RUN make build; \
+    mkdir -p /tmp/build; \
+    cp /go/src/github.com/kubernetes-incubator/descheduler/_output/local/bin/linux/$(go env GOARCH)/descheduler /tmp/build/descheduler
```

Comment:
Why can't we just move here? Also, is there a way to to build the image in /tmp/build with existing hack scripts in instead of building it locally and then copying?

Reply:
@ravisantoshgudimetla I can change the copy to move, but I think its best to be consistent with the other places we've done this:
https://github.com/openshift/console-operator/pull/187
https://github.com/openshift/prom-label-proxy/pull/7
https://github.com/openshift/origin/pull/22410

My hesitation about fixing hack/build-go.sh is that I'm not sure what else I may inadvertently break in the process. 
https://github.com/openshift/descheduler/blob/master/hack/lib/build/binaries.sh#L243

In this case it would probably be safe to make the changes in the hack scripts, since it doesn't look like there is any cross-compilation outside the container builds.

With that said.. I think this fix (admittedly hacky looking) is the safest thing to do.

Reply:
Ok, I am fine with it then, will create some TODO issues later.

/lgtm